helios/templates/election_view.html

```diff
@@ -207,6 +198,55 @@ <h3 class="highlight-box">Tally</h3>
 </span> <br /><br />
 {% endif %}
 
+
+
+
+
+
+
+    </div>
+    <div class="small-12 large-3 columns" style="text-align: left;">
+        <br />
+        <div>
+            <strong>Operations:</strong>
```

Comment:
I put the Operários and Information tag lines, more for user education. Not necessary the most need but.
Also for agreggation... All operations and informations on one place.

And its easy this way to makes things more responsive also... A list of elements its super easy to Go responsive and Works well on all browsers